File: RAI/EjemploBanco/JavaImplementation/Corriente.java
public class Corriente extends Cuenta{
    private int cupoTarjeta;

    public Corriente(int id, int saldo, int cupoTarjeta){
        super(id,saldo);
        this.cupoTarjeta = cupoTarjeta;
    }

    public int getCupoTarjeta(){
        return this.cupoTarjeta;
    }
    
    public String toString(){
        return super.toString() + "cupo: " + this.cupoTarjeta + " ";
    }
}

File: RAI/EjemploBanco/python_implementation/Corriente.py
from Cuenta import Cuenta

class Corriente(Cuenta):

    def __init__(self, id, saldo, cupo_tarjeta):
        super().__init__(id, saldo)
        self.cupo_tarjeta = cupo_tarjeta

    def get_cupo_tarjeta(self):
        return self.cupo_tarjeta

    def __str__(self):
        return super().__str__() + "Cupo: " + str(self.cupo_tarjeta) + " "